src/api/custom-resolvers/queries/get-application-by-reference-number/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import { GetApplicationByReferenceNumberResponse, GetApplicationByReferenceNumberVariables } from '../../../types';
+import getIdsByReferenceNumber from '../../../helpers/get-ids-by-reference-number';
+import getPopulatedApplication from '../../../helpers/get-populated-application';
+import decryptNominatedLossPayee from '../../../helpers/decrypt-nominated-loss-payee';
+
+/**
+ * getApplicationByReferenceNumber
+ * Gets an application by reference number
+ * Based on decrypt variables, decrypts part of application
+ * returns full application
+ * @param {Object} GraphQL root variables
+ * @param {Object} GraphQL variables for the getApplicationByReferenceNumberVariables mutation
+ * @param {Object} KeystoneJS context API
+ * @returns {Promise<GetApplicationByReferenceNumberResponse>} Application with success flag
+ */
+const getApplicationByReferenceNumber = async (
+  root: any,
+  variables: GetApplicationByReferenceNumberVariables,
+  context: Context,
+): Promise<GetApplicationByReferenceNumberResponse> => {
+  try {
+    console.info('Getting application by reference number');
+
+    const { referenceNumber, decryptFinancialUk } = variables;
+
+    // array of ids in application from provided application reference number
+    const ids = await getIdsByReferenceNumber(referenceNumber, context);
+
+    // if object is populated
+    if (ids) {
```

Comment:
`ids` is really confusing, can we just call it `application`?

I.e the `getPopulatedApplication` function's param is called `application`.